features/gezag-minderjarige/gezag-niet-te-bepalen.feature

```diff
@@ -0,0 +1,111 @@
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                                            |
+      | type        | GezagNietTeBepalen                                                                |
+      | toelichting | gezag is niet te bepalen omdat minderjarige niet of tijdelijk in Nederland woont. |
+
+    Scenario: Minderjarige verblijft tijdelijk in Nederland
+      Gegeven de persoon 'Bert' met burgerservicenummer '000000036'
+      * is minderjarig
+      * is ingeschreven in de RNI
+      # En heeft een tijdelijk verblijfsadres in Nederland
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                                            |
+      | type        | GezagNietTeBepalen                                                                |
+      | toelichting | gezag is niet te bepalen omdat minderjarige niet of tijdelijk in Nederland woont. |
+
+    Scenario: Verblijfplaats van minderjarige is onbekend
+      Gegeven de persoon 'Bert' met burgerservicenummer '000000036'
+      * is minderjarig
+      * is ingeschreven in de RNI
+      # heeft een volledig onbekende verblijfplaats
+      Als gezag wordt gezocht met de volgende parameters
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      Dan heeft de response een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000036 |
+      En heeft de persoon een 'gezag' met de volgende gegevens
+      | naam        | waarde                                                                            |
+      | type        | GezagNietTeBepalen                                                                |
+      | toelichting | gezag is niet te bepalen omdat minderjarige niet of tijdelijk in Nederland woont. |
+
+  
+  Regel: Gezag is niet te bepalen als is vastgesteld dat de minderjarige niet meer op het geregistreerde adres woont, dan is het gezag niet te bepalen
+
+    Scenario: Er is vastgesteld dat de minderjarige niet meer op het adres verblijft
+      Gegeven de persoon 'Bert' met burgerservicenummer '000000036'
+      * is minderjarig
+      * is ingeschreven in de BRP
+      En er loopt onderzoek waarin is vastgesteld dat de minderjarige niet op het adres verblijft
```

Comment:
Als er is vastgesteld dat de minderjarige niet op het adres verblijft (zie titel), dan loopt er toch geen onderzoek meer?

Reply:
jawel. Bij afronden van onderzoek kan er één van de volgende twee dingen gebeuren:
1. het nieuwe adres wordt ingevoerd. 
2. het onderzoek wordt gesloten zonder wijzigingen.

In beide gevallen geldt dat de actuele verblijfplaats dan juiste gegevens toont

Reply:
Ik denk dat ik het nog steeds niet begrijp. Volgens mij zeggen we bij personen bevragen dat als de aanduiding is 089999 dat dan is vastgesteld dat de persoon niet op het adres verblijft en hier zeggen we dat er een onderzoek loopt waarin is vastgesteld dat de minderjarige niet op het adres verblijft

Reply:
Ja dat klopt. De indicatieVastgesteldVerblijftNietOpAdres 089999 is een lopend actueel onderzoek. 
Het onderzoek is niet afgesloten. Dit is eigenlijk een tussenresultaat, maar geen afsluiting van het onderzoek. Waar persoon wel woont weten we immers nog niet niet, maar we weten in ieder geval dat de persoon niet op het adres woont waar persoon is ingeschreven.

Reply:
```suggestion
      de gemeente heeft vastgesteld dat de minderjarige niet op het adres verblijft
```

Reply:
Is het duidelijk aan de hand van bovenstaande uitleg en suggestie, @MelvLee ?